zdm/tests/Scattering/test_scattering_methods.py

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
please move this file up one level to `tests`

I will then turn on CI to have the code auto-magically tested.

Reply:
done! I think at least this is what you are after. Will be tough to reduce this to a simple yes/no output but will see.